libs/client/use-cases/dashboard/shared/ui/src/lib/footer/footer.component.html

```diff
@@ -1 +1,6 @@
-<p>footer works!</p>
+<div>
```

Comment:
please use the BEM approach: https://devpebe.com/2019/05/31/metodologia-bem-jak-pisac-dobry-kod-css-praktyczne-przyklady/